digital_asset_types/src/rpc/display_options.rs

```diff
@@ -0,0 +1,9 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, Eq, PartialEq, JsonSchema, Default)]
+#[serde(deny_unknown_fields, rename_all = "camelCase")]
+pub struct DisplayOptions {
+    #[serde(default)]
+    pub show_unverified_collections: bool,
```

Comment:
Including unverified collections is not strictly a "display option" since it changes the underline query. Perhaps we should add `groupVerifiedOnly` top-level parameter to the handlers, which by default is `true` but can be set to `false`. Also, note that "collection" is one type of "group", so we should try to keep the names more generic.

Reply:
We didn't know if all groups would have a form of verification or not. You might also want to see some groups verified, some not.

The key detail here is that we leverage "show_unverified_collections" across multiple endpoints. So `groupVerifiedOnly` is confusing depending on the API.

I'll propose an option:
What if we just rename "displayOptions" to "options" like we discussed on the call?

Reply:
The `options` sounds good.

Reply:
Correct.

Reply:
Addressed